content/_policy/rules-of-use._fr.md

```diff
@@ -7,151 +7,155 @@ permalink: /fr/policy/rules-of-use/
+Le service Login.gov utilise les meilleures pratiques reconnues en matière de sécurité et de protection de la vie privée.
 
-## 2. Votre consentement
+Avant d'utiliser le service Login.gov, vous devez accepter ces conditions de service. Si le service Login.gov modifie ses conditions d'utilisation, vous aurez la possibilité d'accepter ou de refuser les conditions d'utilisation mises à jour lors de votre prochaine connexion. De même, lorsque vous effectuez certaines activités dans le service Login.gov, telles que la fourniture d'informations personnelles pour la première fois, le service Login.gov peut vous demander de confirmer à nouveau votre compréhension de la manière dont il utilise vos informations.
 
-Outre les détails concernant votre consentement énoncés ailleurs dans les présentes conditions, en acceptant ces conditions et en utilisant le service Login.gov, vous convenez de ce qui suit :
+## 2. Votre accord
 
-1. Vous n'êtes pas un enfant de moins de 13 ans,
-2. Toute information que vous nous fournissez est complète et exacte,
-3. Si vous vérifiez votre identité, l'identité que vous revendiquez lorsque vous utilisez le service est la vôtre,
-4. Vous vous conformerez aux lois locales, nationales et fédérales en vigueur lors de l'utilisation du service
-5. Vous garderez en toute confidentialité vos informations personnelles et de connexion, et 6. Vous maintiendrez des informations exactes sur votre compte à tout moment.
+En plus des détails concernant votre consentement mentionnés ailleurs dans ces conditions, en acceptant ces conditions et en utilisant le service Login.gov, vous acceptez que :
 
-Vous vous engagez en outre à NE PAS déformer votre identité ou toute information que vous présentez dans le cadre du service Login.gov, y compris par le biais des canaux d'assistance à la clientèle.
+1. Vous n'êtes pas un enfant de moins de 13 ans,
+2. Toutes les informations que vous nous fournissez sont complètes et exactes,
+3. Si vous vérifiez votre identité, l'identité que vous déclarez lors de l'utilisation du service est la vôtre,
+4. Vous vous conformerez aux lois locales, nationales et fédérales applicables lors de l'utilisation du service,
+5. Vous préserverez la confidentialité de vos données personnelles et de vos informations de connexion, et
+6. Vous conserverez en permanence des informations exactes sur votre compte.
 
-Vous consentez également à être lié par ces conditions et par la [politique de confidentialité]({% link _policy/index._fr.md %}), ainsi que par d'autres conditions en lien avec le service Login.gov.
+Vous acceptez en outre de NE PAS déformer votre identité ou les informations que vous présentez dans le cadre du service Login.gov, y compris par l'intermédiaire des canaux d'assistance à la clientèle.
+Vous acceptez également d'être lié par ces conditions et par la [politique de confidentialité]({% link _policy/index._fr.md %}), ainsi que par d'autres conditions liées au service Login.gov.
```

Comment:
If this is meant to be a separate paragraph, It needs a line before. Currently it shows as one continuous paragraph with the preceding line.

```suggestion

Vous acceptez également d'être lié par ces conditions et par la [politique de confidentialité]({% link _policy/index._fr.md %}), ainsi que par d'autres conditions liées au service Login.gov.
```